site/Dance/styles.css

```diff
@@ -12,10 +12,56 @@ h1 {
   padding: 180px;
   border-style: solid;
   border-color: gray;
+}
+
+@media (max-width: 600px) {
+  h1 {
+    font-size: 70px;
+    color: rgb(88, 88, 88);
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    padding: 180px;
+    border-style: solid;
+    border-color: gray;
```

Comment:
You don't have to change but I just wanted to point out that there are 2 different ways you can do border stuff. One way is how you're doing it where you set each border property individually, so like: 

```
border-style: solid;
border-color: gray;
```

another way to do it is what's called shorthand where you can do those two things at once:

```
border: solid gray;
```

You'll even see people set the width in it as well, like so:

```
border: 1px solid gray;
```

Just wanted to point out. What you have is good!